templates/test/list_table.html

```diff
@@ -17,7 +17,15 @@
                         <td data-title="Details">{{ test.test_type.description }} <a href="{{ test.github_link }}" target="_blank">{{ (test.commit[:7] if test.test_type == TestType.commit else test.pr_nr) }}</a></td>
                         <td data-title="Platform">{{ test.platform.description }}</td>
                         <td data-title="Run Errors">{{ "Unknown" if not test.finished else ("Yes" if test.failed else "No") }}</td>
-                        <td data-title="Completed"><a href="{{ url_for('test.by_id', test_id=test.id) }}">{{ "Yes" if test.finished else "No" }}</a></td>
+                        <td data-title="Completed">
+                            <a href="{{ url_for('test.by_id', test_id=test.id) }}">{{ "Yes" if test.finished else "No" }}</a>
+                            {% if user.is_admin or request.endpoint == 'custom.index' %}
```

Comment:
So any user that manager to get this view rendered from the correct place can restart the test... doesn't seem good.

Reply:
There is a check at backend too, which will check whether the test has been started by the same user or not.
Or other thing we can do is make a function call in test model, which will confirm whether to display start/stop button or not

Reply:
Even if there's a back-end check on the restart_test controller, we still want to hide it here whenever it's not applicable. A function in the test model isn't the correct place. Rather call this view with an extra variable that decides whether or not to show it.